install.go

```diff
@@ -0,0 +1,58 @@
+package helmclient
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	"github.com/prometheus/client_golang/prometheus"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+)
+
+// InstallReleaseFromTarball installs a chart packaged in the given tarball.
+func (c *Client) InstallReleaseFromTarball(ctx context.Context, chartPath, namespace string, values map[string]interface{}, options InstallOptions) error {
+	eventName := "install_release_from_tarball"
+
+	t := prometheus.NewTimer(histogram.WithLabelValues(eventName))
+	defer t.ObserveDuration()
+
+	err := c.installReleaseFromTarball(ctx, chartPath, namespace, values, options)
+	if err != nil {
+		errorGauge.WithLabelValues(eventName).Inc()
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
+
+func (c *Client) installReleaseFromTarball(ctx context.Context, chartPath, namespace string, values map[string]interface{}, options InstallOptions) error {
```

Comment:
For my curiosity, why are the values not options in that sense?

Reply:
The Helm API changed. In Helm 2 there was a ValueOverrides option.

In Helm 3 the install action takes in a chart and values. So I'd like to align with that.

https://github.com/helm/helm/blob/v3.1.0/pkg/action/install.go#L163